lib/pbench/test/unit/server/database/test_index_map_db.py

```diff
@@ -221,3 +243,16 @@ def fake_query(db_type):
         with pytest.raises(IndexMapSqlError) as e:
             IndexMap.exists(drb)
         assert str(e.value) == "Error checkexist index drb:any: That was easy"
+
+    def test_stream_fail(self, monkeypatch, db_session, attach_dataset):
+        """Test index stream failure"""
+
+        def fake_query(db_type):
+            raise SQLAlchemyError("That was easy")
+
+        drb = Dataset.query(name="drb")
+        monkeypatch.setattr(Database.db_session, "query", fake_query)
+
+        with pytest.raises(IndexMapSqlError) as e:
+            [i for i in IndexMap.stream(drb)]
```

Comment:
`i for i` doesn't really accomplish anything...I would have opted for `list(IndexMap.stream(drb))`.

Reply:
Ha.